pkg/middlewares/auth/forward.go

```diff
@@ -214,4 +231,6 @@ func writeHeader(req, forwardReq *http.Request, trustForwardHeader bool) {
 	default:
 		forwardReq.Header.Del(xForwardedURI)
 	}
+
+	forwardReq.Header = filterForwardRequestHeaders(forwardReq.Header, allowedHeaders)
```

Comment:
Accordingly to the previous comment, those lines would be removed.
```suggestion
```

Reply:
ok